draft-ietf-tls-tls13.md

```diff
@@ -2553,7 +2553,7 @@ obfuscated_ticket_age
   configuration that it is using, in milliseconds.  This value is
   added modulo 2^32 to the "ticket_age_add" value that was
   included with the ticket, see {{NewSessionTicket}}.  This addition
-  prevents passive observers from correlating sessions unless tickets
+  prevents passive observers from correlating connections unless tickets
```

Comment:
I'm not sure about this one. Which things precisely are not meant to be correlated by passive observers?